geo/src/algorithm/simplify.rs

```diff
@@ -19,6 +19,10 @@ fn rdp<T>(points: &[Point<T>], epsilon: &T) -> Vec<Point<T>>
 where
     T: Float,
 {
+    // Epsilon must be greater than zero for any meaningful simplification to happen
+    if *epsilon <= T::zero() {
```

Comment:
This implies some kind of programmer error right? Should we add a debug assertion failure here?

Reply:
I'm *assuming* the fuzzer runs against release builds and thus won't trigger the debug assertion? But I might be wrong...

Reply:
cargo-fuzz builds in release mode _with_ debug assertions

https://github.com/rust-fuzz/cargo-fuzz/blob/799bf534b889859b090b76f2ca1cffdca4ee7455/src/main.rs#L45-L47

https://github.com/rust-fuzz/cargo-fuzz/blob/bde819e87a86e6b1c5c0320e94343a6f97d063bb/src/options.rs#L68-L70

i don't know where the default value for debug-assertions gets set in cargo-fuzz, but i verified locally that debug assertions are enabled

Reply:
Yeah, I don't feel confident about how to resolve this.

I can imagine some cases where it would be interesting if it were possible to reach an assert via the fuzzer, and others, like here, where it wouldn't be interesting.

We *could* have the convention that `debug_assert` is the former while `assert` is the latter, but... that seems pretty limiting.

We could make our own assert macros, like `debug_assert_unless_fuzzing!`, but that seems a bit much.

Reply:
To restate, I'm really OK with just merging this as is. 👍

Reply:
I am feeling very indecisive too.

Reply:
+1 to merging as-is